x/sysproxy/sysproxy_windows.go

```diff
@@ -0,0 +1,108 @@
+//go:build windows
+
+package sysproxy
+
+import (
+	"errors"
+	"fmt"
+
+	"golang.org/x/sys/windows"
+	"golang.org/x/sys/windows/registry"
+)
+
+var (
+	modwininet            = windows.NewLazySystemDLL("wininet.dll")
+	procInternetSetOption = modwininet.NewProc("InternetSetOptionW")
+)
+
+const (
```

Comment:
*nit* maybe add a comment of the link to the corresponding Windows API documentation?